### PR TITLE
Add missing toString function

### PR DESCRIPTION
**Pull request by tobiasz135**

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java [hunk 1]
```diff
@@ -29,5 +29,9 @@ public void operateTo(int x, int y) {
             driver.operateTo(x, y);
         }
     }
+    @Override
+    public String toString() {
+        return "Logger + Line Driver";
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/drivers/DriverComposite.java]: It is not generic. The name should be created from the names of drivers in the list!

**Review by kowallus — CHANGES_REQUESTED** [review]
the name shouldn't be fixed

**Review by tobiasz135 — COMMENTED** [review]
Task issue fixed

**Review by kowallus — APPROVED** [review]
thank you

**Pull request merged**